spec/integration/hanami/assets/compiler_spec.rb

```diff
@@ -111,7 +111,7 @@
 
     target  = @config.public_directory.join("assets", "compile-sass.css")
     content = target.read
-    expect(content).to match %(body {\n  font: 100% Helvetica, sans-serif;\n  color: #333; }\n)
+    expect(content).to match %(body {\n  font: 100% Helvetica, sans-serif;\n  color: #333;\n})
```

Comment:
The output looks different because `dart-sass` now defaults to `:expanded` output style instead of `:compact` output style.